examples/quickstart-pytorch/README.md

```diff
@@ -47,6 +47,8 @@ Write the command below in your terminal to install the dependencies according t
 pip install -r requirements.txt
 ```
 
+______________________________________________________________________
+
```

Comment:
I'm thinking we could remove these, since the immediately below section title will introduce one already.

Reply:
sure.